spec/support/spec_config.rb

```diff
@@ -475,11 +478,28 @@ def x509_auth?
     uri_options[:auth_mech] == :mongodb_x509
   end
 
-  # When we use x.509 authentication, omit all of the users we normally create
-  # and authenticate with x.509.
-  def credentials_or_x509(creds)
-    if x509_auth?
-      {auth_mech: :mongodb_x509}
+  # When we authenticate with a username & password mechanism (scram, cr)
+  # we create a variety of users in the test suite for different purposes.
+  # When we authenticate with passwordless mechanisms (x509, aws) we use
+  # the globally specified user for all operations.
+  def fixed_user?
```

Comment:
I don't find the name of this method very clear -- are you differentiating between mechanisms that use global users vs those that make their own? Maybe `global_user?` would be a clearer name?